steps/autify-upload/0.0.2/step.yml

```diff
@@ -0,0 +1,65 @@
+title: |
+  Autify Upload
+summary: |
+  Uploads your build to Autify.
+description: |-
+  Uploads you build `.app` to [Autify](https://autify.com/mobile) using [Autify's API](https://mobile-app.autify.com/api/docs/index.html). You can upload the iOS build with the Step.
+
+  This Step, however, does NOT generate your build: to create a `.app` file, you need the right step. For example, `Xcode build for simulator` Step or any other Step can generate a build
+website: https://github.com/autifyhq/bitrise-step-autify-upload
+source_code_url: https://github.com/autifyhq/bitrise-step-autify-upload
+support_url: https://github.com/autifyhq/bitrise-step-autify-upload/issues
+published_at: 2021-12-14T05:27:01.003204+09:00
+source:
+  git: https://github.com/autifyhq/bitrise-step-autify-upload.git
```

Comment:
Script should not download another executable.
See: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#do-not-use-submodules-or-require-any-other-resource-downloaded-on-demand

Reply:
Valid comment @koral-- , thanks for reviewing it.

It would be better to install dependencies using Bitrise CLI dependency handling mechanism, but it seems `autify-for-mobile-cli` is not available in `brew`.

I do not think this blocks us from merging the Step.